stl/inc/coroutine

```diff
@@ -142,6 +142,7 @@ struct coroutine_handle {
 
     void destroy() const noexcept { // strengthened
         __builtin_coro_destroy(_Ptr);
+        _Ptr = nullptr;

```

Comment:
Given this `destroy` is a `const` member function, do you really want to modify the member?